extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -257,6 +260,30 @@ export class AmpAnalytics extends AMP.BaseElement {
   }
 
   /**
+   * If requestsFrameUrl (and optionally dataHash as well) are specified in
+   * config, check whether third-party iframe already exists, and if not,
+   * create it.
+   * @param {!JSONType} config
+   * @private
+   */
+  handleRequestsFrameUrl_(config) {
+    if (config['requestsFrameUrl'] && config['dataHash']) {
+      if (!config['dataHash'].startsWith('#')) {
+        config['requestsFrameUrl'] += '#';
+      }
+      config['requestsFrameUrl'] += config['dataHash'];
```

Comment:
You're right, this is kind of sketchy. My logic at the time is that the map is going to be keyed off of the concatenation, so might as well not duplicate that (admittedly small) effort every time we do a get/set/has on it. But...yeah, it's wrong. Will fix. Thanks.